src/utils/rates.js

```diff
@@ -109,14 +109,20 @@ export const fiatInvestmentBalance = (balance: BigNumber, rates: RatesPerChain,
   return fiatAmount?.toFixed(2);
 };
 
-export const fiatTokenValue = (tokenValue: number, rates: RatesPerChain, currency: Currency, formatter: any) => {
-  if (isNaN(tokenValue)) return '';
+export const fiatTokenValue = (
+  tokenValueInUSD: number,
+  rates: RatesPerChain,
+  currency: Currency,
+  formatter?: any,
+  decimals?: number,
+) => {
+  if (isNaN(tokenValueInUSD)) return '';
   const nativeAssetRate = rates?.[nativeAssetPerChain[CHAIN.ETHEREUM].address];
   const currencySymbol = getCurrencySymbol(currency);
 
-  const fiatAmount = (tokenValue * nativeAssetRate?.[currency]) / nativeAssetRate?.[USD];
+  const fiatAmount = (tokenValueInUSD * nativeAssetRate?.[currency]) / nativeAssetRate?.[USD];
```

Comment:
What happens if `nativeAssetRate` is `null`?

Reply:
Got (currency symbol)0. like $0

Reply:
Thanks for checking